src/main/java/com/ceritoxi/util/Math.java

```diff
@@ -1,130 +1,457 @@
 package com.ceritoxi.util;
+import java.util.Stack;
 
+public class Meth {
+
+    public static boolean isDividableByXorY(int dividend, int x, int y) {
+        if ((x != 0 && dividend % x == 0) || (y != 0 && dividend % y == 0)) {
+            return true;
+        }
+        return false;
+    }
+
+    public static int sumOfNumbersUnderZThatAreDividableByXorY (int x, int y, int z) {
+        int sum = 0;
+
+        if (z < 1) {
+            return sum;
+        }
+        for (int i = 1; i < z; i++) {
+            if (isDividableByXorY(i, x, y)) {
+                sum += i;
+            }
+        }
+        return sum;
+    }
+
+    public static boolean isEven (int number) {
+        if (number % 2 == 0) {
+            return true;
+        }
+        return false;
+    }
+
+    public static boolean isEven (long number) {
+        if (number % 2 == 0) {
+            return true;
+        }
+        return false;
+    }
+
+    public static int sumOfEvenFibonacciTermsUnderX (int x) {
+
+        int result = 0;
+        int sum = 0;
+        int firstTerm = 0;
+        int secondTerm = 1;
+
+        while (sum < x) {
+            if (isEven(sum)) {
+                result += sum;
+            }
+            sum = firstTerm + secondTerm;
+            firstTerm = secondTerm;
+            secondTerm = sum;
+        }
+        return result;
+    }
+
+    public static boolean isPrime (long number) {
+
+        if (number == 2 || number == 3 || number == 5) {
```

Comment:
I live prime check since the topic goes so many places of maddening programming problems :). so minor performance note: Look up auto-casting and auto-boxing in java. I don't want to put a wall of text here but right here you compare number (a long value) with ```2``` which is an int value. While java magic is okay with this (most of the times), it still have to make a "cast". Since it cannot compare apples with oranges it will cast 2 into a long. This is a plus operation which can be spared by declaring 2 like ```2L``` telling explicitly that its a long.

Reply:
Sure :D